scripts/build.sh

```diff
@@ -241,8 +241,8 @@ case "${1}" in
 		do_assets
 		;;
 
-	docker-dev)
-		do_docker_dev_image
```

Comment:
We should avoid dead code and remove the `do_docker_dev_image` function

Reply:
Fixed